src/plugins/auto_batch/src/plugin.cpp

```diff
@@ -24,22 +20,18 @@ namespace autobatch_plugin {
 
 std::vector<std::string> supported_configKeys = {CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG),
```

Comment:
let's keep it and remove once we remove old API and `supported_configKeys` at all.
In order to avoid compiler warnings - let's wrap with macro `OPENVINO_SUPPRESS_DEPRECATED_START` / `OPENVINO_SUPPRESS_DEPRECATED_END`